cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -21,6 +21,11 @@
 #include "cartographer_ros/occupancy_grid.h"
 #include "cartographer_ros_msgs/msg/trajectory_submap_list.hpp"
 
+#include "rclcpp/clock.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp/time.hpp"
+#include "rclcpp/time_source.hpp"
```

Comment:
Nit: include style inconsistent with the rest of the file.
Nit: include order, this should go below nav_msgs

same below